uk.co.compucorp.civicrm.tasksassignments/CRM/Tasksassignments/Upgrader.php

```diff
@@ -210,7 +210,7 @@ public function upgrade_0007() {
         ),
         'keydates_tab' => array(
           'label' => 'Show or hide the Key Dates tab',
-          'value' => 1,
+          'value' => 2,
```

Comment:
Sorry, forgot to explain this even though I was asking @davialexandre earlier :man_facepalming: The problem is that there's a collision with the upgrader values. @davialexandre fixed this in another branch: https://github.com/compucorp/civihr-tasks-assignments/commit/dda35eac87a0e44e6c6fbd0ccf916f5ee6ef1b4e but I needed this fix, otherwise tests would not run.